core/src/main/kotlin/ch/ergon/dope/resolvable/fromable/Bucket.kt

```diff
@@ -17,3 +18,11 @@ class AliasedBucket(name: String, val alias: String) : Bucket(name) {
 }
 
 fun AliasedBucket.field(field: String): Field<StringType> = Field(field, alias)
+
+fun Bucket.all(): Expression = Field<StringType>(
+    "*",
```

Comment:
This Asterisk symbol is used in several places now, maybe it makes sense to define it in a global constant

Reply:
added new AsteriskExpression